### PR TITLE
Cleanup of LoginRequired

### LINKED ISSUE #9446: Cleanup of LoginRequired component
## Test plan

https://github.com/brave/browser-laptop/pull/9493#issue-236177908

----

**Describe the issue you encountered:**
Cleanup of `LoginRequired` component, preparation for converting `Main` into redux component.

### PR DESCRIPTION
**Pull request by NejcZdovc**
Submitter Checklist:

- [x] Submitted a [ticket](https://github.com/brave/browser-laptop/issues) for my issue if one did not already exist.
- [x] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [ ] Added/updated tests for this change (for new code or code which already has tests).
- [x] Ran `git rebase -i` to squash commits (if needed).
- [x] Tagged reviewers and labelled the pull request [as needed](https://github.com/brave/browser-laptop/wiki/Pull-request-process).

Resolves #9446

Auditors: @bsclifton

Test Plan:
- go to https://login.mozilla.com and check if login is displayed correctly

Reviewer Checklist:

Tests

- [ ] Adequate test coverage exists to prevent regressions
- [ ] Tests should be independent and work correctly when run individually or as a suite [ref](https://github.com/brave/browser-laptop/wiki/Code-Guidelines#test-dependencies)
- [ ] New files have MPL2 license header

### CODE REVIEW

File: app/renderer/components/main/loginRequired.js [hunk 1]
```diff
@@ -133,26 +133,23 @@ class LoginRequired extends React.Component {
...
-          <Button l10nId='ok' className='primaryButton' onClick={this.onSave.bind(this)} />
+          <Button l10nId='ok' className='primaryButton' onClick={this.onSave} />
         </CommonFormButtonWrapper>
       </CommonForm>
     </Dialog>
   }
 }
 
-LoginRequired.propTypes = { frameProps: PropTypes.object }
+module.exports = LoginRequired
```
Comment (bbondy) [inline app/renderer/components/main/loginRequired.js]: nit: I prefer this at the end of a file

Comment (NejcZdovc) [inline app/renderer/components/main/loginRequired.js]: In all components we have this like that, so I would leave it for now and we can move it all to the bottom in a different PR

Comment (bbondy) [inline app/renderer/components/main/loginRequired.js]: ok I'm not concerned either way, was just noting.

Comment (luixxiul) [inline app/renderer/components/main/loginRequired.js]: I noticed it as well and was trying to re-move the change.

Comment (luixxiul) [inline app/renderer/components/main/loginRequired.js]: @NejcZdovc please document the rule on the wiki page for refactoring work, thanks. CC @cezaraugusto

**Pull request merged**